### ISSUE TITLE
common adaptor should use new expandReferences API

### ISSUE THREAD

**Issue by josephjclark** [OpenFn/adaptors#306]
To be done urgently

**Comment by josephjclark** [comment]
Adding this to the HTTP epic - although really it's a common 2.0 epic.

Since we're about to bump to common 2.0, we should do this.

Just like in #315, this will break compatibility with many adaptors. But semver will protect us from the worst of the damage, and the the worst-case scenario fix is to put out a patch release with an updated package json.

I think it's important that common 2.0 has a nice clean code base, because it's going to set the standard across the repo.

**Comment by josephjclark** [comment]
Common is not going to be bumped to 2.0 right now. I'm reducing the priority on this.

**Comment by josephjclark** [comment]
I don't think there's anything left to do here? Maybe someone can investigate?

There are 3 closely related issues: I'd love to close, merge or fix them together: see also #317 and #307

**Comment by martalovescoffee** [comment]
@josephjclark Can you provide a bit more information on what should be investigated. It's a bit hard to understand.

**Comment by aleksa-krolls** [comment]
@hunterachieng this issue still need estimates

**Comment by josephjclark** [comment]
Yes this is mess 😅  We actually need to consolidate all 3 issues into one, and scope the exact work that's needed next.

If @hunterachieng is estimating can we book a call to discuss and tidy this up?

**Comment by hunterachieng** [comment]
@josephjclark  yes we can discuss it. I have removed the estimate

**Comment by martalovescoffee** [comment]
1h for making the estimate and new issue with actual dev estimate.
